release/util/download_wheels.sh

```diff
@@ -35,7 +35,7 @@ download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERS
 
 # macOS.
 echo "Downloading Ray core MacOS wheels (x86_64)"
-download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37-macosx_10_15_x86_64.whl"
+download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37m-macosx_10_15_x86_64.whl"
```

Comment:
do you know why only cp37 has this m character?

Reply:
The `m` means it's for Python 3.7 with the pymalloc extension

https://stackoverflow.com/questions/62008090/whats-the-different-between-python-abi-cp37-and-cp37m

and it is the version used in the manylinux wheel building container image.

Reply:
but @rickyyx explicitly removed that when adding these. and I asked about the change.

Reply:
ah, sorry, i think i was wrong when removing it for 3.7.